### PR TITLE
Added Tokemak to Dune Spells and added list of relevant Tokemak tokens. - Ready for review

### PR DESCRIPTION
**Pull request by addmorebass**
Added Tokemak as a project and added the first Tokemak spell.

### CODE REVIEW

**Comment by addmorebass** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by addmorebass** [comment]
recheck

**Comment by addmorebass** [comment]
recheck

File: models/tokemak/ethereum/tokemak_ethereum_view_tokemak_lookup_reactors.sql [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            "Tokemak",
+             \'["addmorebass"]\') }}'
+) }}
+
+WITH tokemak_ethereum_view_tokemak_lookup_reactors
+(
+	    reactor_address, underlyer_address, reactor_name, is_deployable
+) AS (
+
+    SELECT '0xD3B5D9a561c293Fb42b446FE7e237DaA9BF9AA84' as reactor_address, '0xdBdb4d16EdA451D0503b854CF79D55697F90c8DF' as underlyer_address, 'ALCX Reactor', true
```
Comment (Hosuke) [inline models/tokemak/ethereum/tokemak_ethereum_view_tokemak_lookup_reactors.sql:15]: We can use varbinary type for hexstring addresses.
```suggestion
    SELECT 0xD3B5D9a561c293Fb42b446FE7e237DaA9BF9AA84 as reactor_address, 0xdBdb4d16EdA451D0503b854CF79D55697F90c8DF as underlyer_address, 'ALCX Reactor', true
```

**Review by Hosuke — CHANGES_REQUESTED** [review]
Hi @addmorebass,
thank you for your contribution.

For the static spell, we may use the similar format to this:
https://github.com/duneanalytics/spellbook/blob/main/models/tokens/arbitrum/tokens_arbitrum_erc20.sql

File: models/tokemak/ethereum/tokemak_ethereum_view_tokemak_lookup_reactors.sql [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{{ config(
+    tags=['dunesql'],
```
Comment (Hosuke) [inline models/tokemak/ethereum/tokemak_ethereum_view_tokemak_lookup_reactors.sql:2]:
```suggestion
    tags=['dunesql', 'static'],
```

File: models/tokemak/ethereum/tokemak_ethereum_view_lookup_tokens.sql [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+    schema='tokemak_ethereum',
+    alias = alias('view_lookup_tokens'),
+    post_hook = '{{ expose_spells(\'["ethereum"]\',
+        "project", 
+            "Tokemak",
+             \'["addmorebass"]\') }}'
+) }}
+
+
+WITH tokemak_ethereum_view_lookup_tokens(symbol, display_name, address, pricing_contract, decimals, is_pool, is_active, is_liability, is_dollar_stable) AS
```
Comment (Hosuke) [inline models/tokemak/ethereum/tokemak_ethereum_view_lookup_tokens.sql:11]: Hi @addmorebass,

I have transform tokemak_ethereum_view_tokemak_lookup_reactors.sql into `select ... from (values ...)` format:
https://github.com/duneanalytics/spellbook/blob/3456338155e4b868a00c1fe244301b153aea08fe/models/tokemak/ethereum/tokemak_ethereum_view_tokemak_lookup_reactors.sql

You can follow this format to transform this spell as well.

**Review by Hosuke — CHANGES_REQUESTED** [review]

File: models/tokemak/ethereum/tokemak_ethereum_view_lookup_tokens.sql [hunk 4]
```diff
@@ -0,0 +1,233 @@
+{{ config(
+    tags=['dunesql', 'static'],
+    schema='tokemak_ethereum',
+    alias = alias('view_lookup_tokens'),
```
Comment (Hosuke) [inline models/tokemak/ethereum/tokemak_ethereum_view_lookup_tokens.sql:4]: Maybe we can shorten the alias?
```suggestion
    alias = alias('lookup_tokens'),
```

**Comment by addmorebass** [comment]
Thanks for the help @Hosuke - we should be good to go now?

**Comment by addmorebass** [comment]
谢谢

**Comment by Hosuke** [comment]
> Thanks for the help @Hosuke - we should be good to go now?

I will fix the schema and sync the new alias first.

File: models/tokemak/ethereum/tokemak_ethereum_lookup_tokens.sql
Comment (Hosuke) [inline models/tokemak/ethereum/tokemak_ethereum_lookup_tokens.sql:1]: Maybe we can use `csv upload` function for this file in the future.

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @addmorebass

**Review by jeff-dude — APPROVED** [review]
removed the source file, as i believe no sources are used & the listed object was a spell `tokemak_ethereum_lookup_tokens`, which should be a `{{ref()}}` in spellbook instead

**Pull request merged**